README.md

```diff
@@ -65,12 +62,5 @@ To deploy a Tahoe Beta VM with Orka Desktop:
   ghcr.io/macstadium/orka-images/tahoe:latest
 ```
 
-To deploy a Tahoe Beta VM with SIP disabled using Orka Desktop:
-
-1. Click + Create New VM button
-1. Select 'Pull from OCI registry'
-1. Name VM and set parameters (CPUs, Memory, HD size)
-1. In the 'OCI Image Name' field enter: `ghcr.io/macstadium/orka-images/tahoe:latest-no-sip`
-```sh  
-  ghcr.io/macstadium/orka-images/tahoe:latest-no-sip
-```
+> [!NOTE]
+> MacOS 26 (Tahoe) is currently only supported for guest VMs running on Orka version 3.5.0 and above. We aim to support Tahoe host images in the release of Orka 3.6.0.
```

Comment:
What do we mean by "Tahoe host images" btw?

For Desktop users - can't they run Tahoe VMs as long as they are running macOS 15.5 or newer?

Reply:
Fair point, removed the end of that sentence to avoid confusion.